web/src/engine/websites/ScanVFOrg.ts

```diff
@@ -0,0 +1,34 @@
+import { Tags } from '../Tags';
+import icon from './ScanVFOrg.webp';
+import { type Chapter, DecoratableMangaScraper, Page } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import { FetchCSS } from '../platform/FetchProvider';
+
+function MangaLabelExtractor(element: HTMLElement): string {
+    return element.textContent.trim().replace(/Scan VF$/i, '').trim();
+}
+
+@Common.MangaCSS(/^{origin}\/manga\/[^/]+$/, 'div.card div.row h1', MangaLabelExtractor)
+@Common.MangasMultiPageCSS('/manga?page={page}', 'div.series a.link-series')
+@Common.ChaptersSinglePageCSS('div.chapters-list a', Common.AnchorInfoExtractor(false, 'div.text-muted, div.rate-value'))
+@Common.ImageAjax()
+export default class extends DecoratableMangaScraper {
+
+    public constructor() {
+        super('scanvforg', 'ScanVF.org', 'https://scanvf.org', Tags.Media.Manhwa, Tags.Media.Manhua, Tags.Language.French, Tags.Source.Scanlator);
+    }
+
+    public override get Icon() {
+        return icon;
+    }
+
+    public override async FetchPages(chapter: Chapter): Promise<Page[]> {
```

Comment:
May consider using partitioning algorithm such as divide and conquer to perform head requests and find the response that was redirected to url with parameter `bypass=1`, which would be the last page number.
`fetch('https://scanvf.org/scan/705/170', { method: 'HEAD' });`

Than generate all page links and implement `FetchImage` accordingly to extract the image from the HTML page.